library/decimal.po

```diff
@@ -147,14 +147,16 @@ msgstr ""
 
 #: ../../library/decimal.rst:125
 msgid "Quick-start tutorial"
-msgstr ""
+msgstr "快速入門教學"
 
 #: ../../library/decimal.rst:127
 msgid ""
 "The usual start to using decimals is importing the module, viewing the "
 "current context with :func:`getcontext` and, if necessary, setting new "
 "values for precision, rounding, or enabled traps::"
 msgstr ""
+"使用 decimal 的通常起始步驟是匯入模組、使用 :func:`getcontext` 檢視目前的上下文，"
```

Comment:
```suggestion
"使用 decimal 的通常起始步驟是引入模組、使用 :func:`getcontext` 檢視目前的情境，"
```

Reply:
+1, please keep in mind of the translation consistency across different `*.po` files